src/index.html

```diff
@@ -11,7 +11,194 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="form__field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="form__field">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="form__field">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+
+        <div class="form__field">
+          <label for="dateOfBirth">Full date of birth:</label>
+          <input
+            name="dateOfBirth"
+            type="date"
+            id="dateOfBirth"
+          >
+        </div>
+
+        <div class="form__field">
+          <label for="terms">I accept the term of the agreement</label>
+          <input
+            name="terms"
+            type="checkbox"
+            id="terms"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="form__field">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="form__field">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="8"
+            maxlength="15"
+            required
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>ِAn interesting fact about you!</legend>
+
```

Comment:
you seem to have a strange artefact that might prevent visual tests to succeed
<img width="480" alt="Monosnap HTML Form 2022-11-01 11-55-09" src="https://user-images.githubusercontent.com/39308717/199207771-3b744707-541d-4801-8bbe-7fd66f668892.png">

```suggestion
        <legend>An interesting fact about you!</legend>

```

Reply:
Thank you. I did the suggested change and passed the visual tests.